PyMPDATA/options.py

```diff
@@ -73,7 +73,10 @@ def dtype(self):
 
     @property
     def n_iters(self) -> int:
-        """number of iterations (1: upwind, 2: upwind + one corrective iteration, ...)"""
+        """Number of iterations in the MPDATA algorithm
+        (1: upwind, 2: upwind + one corrective iteration, ...).
+        Bigger values mean more accurate results, but also more computational cost.
```

Comment:
should we elaborate what we mean by accuracy here? (e.g., to clarify how it differs from the third-order-terms option, which is controlled by the `third_order_terms` option). Perhaps also worth to refer to a reference?

Reply:
Changed to "smaller errors". Added to reference to third_order_terms.